tests/cryptomatte_tests.py

```diff
@@ -524,19 +524,27 @@ class CryptomatteSetup(unittest.TestCase):
     def setUp(self):
         self.output_file_name = os.path.join(tempfile.gettempdir(), "result.exr")
 
+
         ai.AiBegin()
-        ai.AiMsgSetConsoleFlags(ai.AI_LOG_NONE)
-        ai.AiMsgSetConsoleFlags(ai.AI_LOG_WARNINGS | ai.AI_LOG_ERRORS)
+        self.universe = ai.AiUniverse()
+
+        msg_flags = ai.AI_LOG_WARNINGS|ai.AI_LOG_ERRORS|ai.AI_LOG_PROGRESS
```

Comment:
Why not add in info and maybe debug messages? They might end up being useful when a test fails?

Reply:
Good point, thanks.